### DISCUSSION TITLE
Error Renaming

### DISCUSSION THREAD

**Discussion by wplatt2** [seanap/Plex-Audiobook-Guide#96] [Q&A] [answered]
I get this error when attempting the "Rename Relocate Extras Title" action.

![image](https://user-images.githubusercontent.com/60993083/224562153-a9284d3f-96d8-4591-a8b7-cb19f6223a04.png)

I have edited the .mta files as directed.

Not sure where my issue is coming from.

**Comment by seanap** [comment, marked as answer]
It looks like you messed something up when editing the mta files. Did you double backslash `\\` ?  What does your action mte looks like?  Should be something like this 
![image](https://user-images.githubusercontent.com/17012946/224565145-021a5bd0-7b64-4daf-955e-f35de616707d.png)

**Reply by wplatt2** [reply]
That was it.  I had a single \ instead of the double.

Thanks for the help!
